elements/pfe-jump-links/demo/index.html

```diff
@@ -110,38 +76,12 @@ <h4 slot="pfe-jump-links-nav--heading">Jump to section</h4>
       </pfe-jump-links-nav>
     </div>
     <div class="pfe-l-grid__item pfe-m-12-col pfe-m-9-col-on-lg">
-      <pfe-jump-links-panel class="special" pfe-c-scrolltarget="jumplinks1">
+      <pfe-jump-links-panel pfe-c-offset="20" class="special" pfe-c-scrolltarget="jumplinks1">
```

Comment:
This attribute actually does something, we should leave it. You can also put a css value on the panel `--pfe-jump-links-panel--offset`. It will read css if it's there, and if both are there it will use the attribute.

Reply:
not sure what you mean? I've added it here :) 
It's also [in the demo CSS](https://github.com/patternfly/patternfly-elements/pull/974/files#diff-249d6b2106f2ab730a3828419f373227R18); is that duplicative?

Reply:
@Djfaucette  ^^